frontend/src/components/projectCreate/projectCreationMap.js

```diff
@@ -61,56 +63,48 @@ const ProjectCreationMap = ({ mapObj, setMapObj, metadata, updateMetadata, step,
   });
 
   useLayoutEffect(() => {
-    setMapObj({
-      ...mapObj,
-      map: new mapboxgl.Map({
-        container: mapRef.current,
-        style: MAP_STYLE,
-        center: [0, 0],
-        zoom: 1,
-        attributionControl: false,
-      })
-        .addControl(new mapboxgl.AttributionControl({ compact: false }))
-        .addControl(new MapboxLanguage({ defaultLanguage: locale.substr(0, 2) || 'en' }))
-        .addControl(new mapboxgl.ScaleControl({ unit: 'metric' })),
-    });
+    if (MAPBOX_TOKEN) {
+      setMapObj({
+        ...mapObj,
+        map: new mapboxgl.Map({
+          container: mapRef.current,
+          style: MAP_STYLE,
+          center: [0, 0],
+          zoom: 1,
+          attributionControl: false,
+        })
+          .addControl(new mapboxgl.AttributionControl({ compact: false }))
+          .addControl(new MapboxLanguage({ defaultLanguage: locale.substr(0, 2) || 'en' }))
+          .addControl(new mapboxgl.ScaleControl({ unit: 'metric' }))
+          .addControl(new MapboxGeocoder({
+              accessToken: MAPBOX_TOKEN,
+              mapboxgl: mapboxgl,
```

Comment:
Add more 2 lines to the Geocoder config:
marker: false,
language: locale.substr(0, 2) || 'en'